mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -134,14 +143,9 @@ def load(context, name, playlist_slice=slice(0, None)):
     - MPD 0.17.1 does not fail if the specified range is outside the playlist,
       in either or both ends.
     """
-    uri = context.lookup_playlist_uri_from_name(name)
-    playlist = uri is not None and context.core.playlists.lookup(uri).get()
-    if not playlist:
-        raise exceptions.MpdNoExistError('No such playlist')
-
-    with warnings.catch_warnings():
-        warnings.filterwarnings('ignore', 'tracklist.add.*"tracks".*')
-        context.core.tracklist.add(playlist.tracks[playlist_slice]).get()
```

Comment:
We might as well keep adding the track ignoring warnings if we are looking up the playlist in a way which gets full tracks. Unless we switch to just getting refs like the comment above mentions.

Reply:
The `track` variable here is actually full `Track` objects, but I guess the point of this PR is to do a new lookup of the tracks on `load` and `listplaylistinfo` in case the backend doesn't provided complete information in the returned `Track` objects.

:+1: